packages/components/psammead-caption/package.json

```diff
@@ -18,6 +18,7 @@
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-caption/README.md",
   "dependencies": {
     "@bbc/gel-foundations": "^1.2.0",
+    "@bbc/psammead-paragraph": "^1.0.3",
```

Comment:
This should be moved into `devDependencies`, since it's only used for the Storybook story, not in the source file.